sketch_oct9a.ino

```diff
@@ -99,19 +102,19 @@ int8_t radiansToCappedDegrees(float rad) {
 }
 
 extern "C" {
-void SERCOM1_Handler(void) {
-  imuI2C.onService();
-}
-
 void SERCOM0_Handler(void) {
   gpsUart.IrqHandler();
 }
 
 #if CAPSULE == 2
-void SERCOM3_Handler(void) {
+void SERCOM1_Handler(void) {
   humidityI2C.onService();
 }
 #endif
+
+void SERCOM3_Handler(void) {
```

Comment:
why was this moved?

Reply:
I reordered them so that they're still in numerical order of the SERCOMs. I suppose I didn't have to.